source/ocsp-support/ocsp-support.rst

```diff
@@ -134,6 +134,42 @@ Drivers SHOULD validate OCSP Responses in the manner specified in `RFC
 6960: 3.2 <https://tools.ietf.org/html/rfc6960#section-3.2>`__ to the
 extent that their TLS library allows.
 
+Suggested OCSP Caching Behavior
+-------------------------------
+Drivers with sufficient control over their TLS library's OCSP
+behavior SHOULD implement an OCSP cache. The key for this cache
+SHOULD be the certificate identifier (CertID) as specified in
```

Comment:
nit: change to "certificate identifier (CertID) of the OCSP request"

Reply:
Done!